Uchu.Core/Replicas/ReplicaPacket.cs

```diff
@@ -6,11 +6,21 @@ namespace Uchu.Core
 {
     public class ReplicaPacket : IReplica
     {
+        private string _name;
+        
         public long ObjectId { get; set; }
 
         public int LOT { get; set; }
 
-        public string Name { get; set; }
+        /// <summary>
+        /// Name of the Replica, without newline.
+        /// </summary>
+        public string Name
+        {
+            get => _name;
+            // TODO: Look into this.
+            set => _name = value.Replace((char) 13, '\r');
```

Comment:
what's the reason for using `\r` instead of `\n` here?

Reply:
NPC names have newlines in them for some reason, which translates into a weird nametag. This implementation is although flawed and I'll have to look into why the NPC names have these newlines.